docs/content_management/configure_ct_field_settings.md

```diff
@@ -0,0 +1,99 @@
+---
+description: Configure Field settings in Content Types.
+---
+
+# Configure Content Type Fields
+
+When you [modify a Content Type](create_edit_content_types.md) and add or modify [Fields](content_model.md#fields-and-field-types), 
+you must configure different settings that control the way [[= product_name =]] 
+treats the contents of each Field. Apart from general settings, certain Field Types, for example, [Page](#default-configuration-of-pages) and [Relation](#content-relation-settings) Field Types may have additional settings.
+
+You can only create or modify Content Types when your [user role](../permission_management/work_with_permissions.md) has the `ContentType/Create` or `ContentType/Update` permission.
+
+To modify Field settings, in the **Field definitions** section, expand the **Landing page** header. 
+When done, click **Save** to apply changes to the Content Type.
+
+## General settings
+
+Depending on their type, Fields can have different combinations of the following 
+general settings. 
+
+|Setting|Description|Use|
+--------|-----------|---|
+|Name|A user-friendly name that describes the Field, used in the interface. It can be up to 255 characters long and consist of letters, digits, spaces and special characters.|Required|
```

Comment:
```suggestion
|Name|A user-friendly name that describes the Field, used in the interface. It can be up to 255 characters long and can only consist of letters, digits, spaces and special characters.|Required|
```